### ISSUE TITLE
Incorrect Issue Count

### ISSUE THREAD

**Issue by jasmineflorencio** [ossn/fixme#30]
When you look at the [projects](https://fixme.ossn.club/projects), some of the projects show 0 issues. For example, MediaWiki's issues aren't directly on GitHub but listed on a separate [site](https://phabricator.wikimedia.org/) instead. It looks like Fixme only uses GitHub issues to determine the issue count.

**Comment by alexdor** [comment]
@jasmineflorencio you are right, currently, fixme only uses GitHub issues. @bacharakis any thoughts?

**Comment by tabshaikh** [comment]
@alexdor Okay

**Comment by siddhant1** [comment]
We have to think of another ways to detect issues.

**Comment by alexdor** [comment]
Yeah it would be a good idea, but considering that only an admin can add projects to be parsed this is probably a low priority issue and the biggest part of this issue is from the backend's side
